apps/client/src/components/heading/heading.tsx

```diff
@@ -21,19 +21,21 @@ export const Heading: React.FC<Props> = ({ title, isLogin }) => {
   return (
     <>
       <div className={styles.heading}>
-        <div className={styles.title}>
-          <h1>{title}</h1>
-        </div>
-        <div className="statebutton">
-          <RecordState />
+        <div className={styles.headingContainer}>
+          <div className={styles.title}>
+            <h1>{title}</h1>
+          </div>
+          <div className="statebutton">
+            <RecordState />
+          </div>
         </div>
         <Tooltip anchorSelect=".statebutton" place="bottom">
           מצב שתיות נוכחי
         </Tooltip>
         {!isLogin ? (
           <Login
             sx={{
-              position: 'sticky',
+              position: 'fixed',
```

Comment:
I am not sure what's the user case, but be careful when using `position: fixed` I suggest you test that it works when changing screen size (if you didn't already).